include/seqan3/utility/type_list/traits.hpp

```diff
@@ -114,6 +114,33 @@ auto reverse(type_list<head_t, pack_t...>)
     return concat(reverse(type_list<pack_t...>{}), type_list<head_t>{});
 }
 
+//!\brief Constructs the set difference `list1 \ list2` [recursion anchor]
```

Comment:
```suggestion
//!\brief Constructs the multiset difference `list1 \ list2` [recursion anchor]
```

The same type might occur multiple times in the list